packages/botonic-core/index.d.ts

```diff
@@ -4,3 +4,7 @@ export interface Route {
   text?: RegExp | string | ((input: string) => boolean);
   action: React.ReactNode;
 }
+
+export declare function humanHandOff(session: any, queue_name: string, on_finish: {payload?: any, path?: any}): Promise<void>;
+
```

Comment:
Can we use the Interface Session here too?

Reply:
same